src/reducers/notificationsReducer.ts

```diff
@@ -1,21 +1,69 @@
-export enum ActionTypes {
-  ALERT_SHOW = "ALERT_SHOW",
+export interface Notification {
+  id: number;
+  message: string;
+  timestamp: Date;
 }
 
-export interface INotificationsState {
-  alert: string;
+export type NotificationState = Notification[];
+
+export type NotificationAction = ShowNotification | DismissNotification;
+
+export interface ShowNotification {
+  type: "Notification/Show";
+  message: string;
```

Comment:
lets use payload. we can start defining payload structures, probably a good idea.  I haven't been doing that so far